samples/mount_roloopbackfs/mount.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"flag"
+	"github.com/jacobsa/fuse/samples/roloopbackfs"
```

Comment:
Here and in the other files, can you please separate standard library imports from external imports? In this case, move the `github.com/jacobsa/fuse/samples/roloopbackfs` line after line 24, so that you have 2 groups of imports separated by a blank line.